src/main/java/organice/logic/commands/EditCommand.java

```diff
@@ -185,13 +262,21 @@ public EditPersonDescriptor(EditPersonDescriptor toCopy) {
             setNric(toCopy.nric);
             setName(toCopy.name);
             setPhone(toCopy.phone);
+            setAge(toCopy.age);
+            setPriority(toCopy.priority);
+            setBloodType(toCopy.bloodType);
+            setDoctorInCharge(toCopy.doctorInCharge);
+            setOrganExpiryDate(toCopy.organExpiryDate);
+            setOrgan(toCopy.organ);
+            setTissueType(toCopy.tissueType);
```

Comment:
Uh try not to use setter methods in the constructor, its bad practice. Can read more [here](https://stackoverflow.com/questions/16167637/setter-method-in-constructor)